audit-connector/src/main/java/com/sequenceiq/cloudbreak/audit/converter/AuditEventBuilderUpdater.java

```diff
@@ -0,0 +1,12 @@
+package com.sequenceiq.cloudbreak.audit.converter;
+
+import com.cloudera.thunderhead.service.audit.AuditProto;
+import com.sequenceiq.cloudbreak.audit.model.EventData;
+
+public interface AuditEventBuilderUpdater<T extends EventData> {
+
+    void updateAuditEventBuilderWithEventData(AuditProto.AuditEvent.Builder auditEventBuilder, T source);
```

Comment:
"WithEventData" could be omitted
moreover, `update` would be adequate

Reply:
pls see below my comment regarding to this